src/gui/Application.cpp

```diff
@@ -162,6 +142,31 @@ Application::~Application()
     }
 }
 
+void Application::applyTheme()
+{
+    QString appTheme = config()->get(Config::GUI_ApplicationTheme).toString();
+    if (appTheme == "auto") {
+        if (osUtils->isDarkMode()) {
+            setStyle(new DarkStyle);
+            m_darkTheme = true;
+        } else {
+            setStyle(new LightStyle);
+        }
+    } else if (appTheme == "light") {
+        setStyle(new LightStyle);
+    } else if (appTheme == "dark") {
+        setStyle(new DarkStyle);
+        m_darkTheme = true;
+    } else {
+        // Classic mode, only check for dark theme when not on Windows
+#ifndef Q_OS_WIN
+        m_darkTheme = osUtils->isDarkMode();
+#endif
```

Comment:
I just needed a moment to remember why we are ifdefing this on Windows. You should add a note about Qt not supporting Windows dark mode (yet). I should have documented that better when I wrote this line.